ui/pages/confirmations/components/confirm/info/typed-sign/typed-sign.test.tsx

```diff
@@ -177,7 +177,7 @@ describe('TypedSignInfo', () => {
       type: TransactionType.signTypedData,
       chainId: '0x5',
     });
-    (snapUtils.isSnapId as jest.Mock).mockReturnValue(false);
+    (snapUtils.isSnapId as unknown as jest.Mock).mockReturnValue(false);
```

Comment:
?

Reply:
There's a type error now that the function is written in typescript